design/multi-factor-admin-and-step-up.md

```diff
@@ -171,145 +160,117 @@ For multi-token support in keycloak, when an authentication execution uses crede
 
 ### Authentication section in the admin console
 
 #### Default Flows
 
-The schema described in the logic would be implemented
-in the _Authentication > flows_ section. Some work can be done on the UX to make
-the process more intuitive, but this will be proposed at a later state in the
-design process.
+The schema described in the logic would be implemented in the _Authentication > flows_ section. Some work can be done on the UX to make
+the process more intuitive, but this is not covered by this design.
 
-Keycloak should have the same default flows as is currently the case, except that two
-authentication factors are used, with the following flow for the browser case:
-```
-Auth type                         | Requirement
------------------------------------------------------------------------------------------------
-Auto                               [x] Alternative  [ ] Required  [ ] Conditional  [ ] Disabled
-  | - Cookie                       [x] Alternative  [ ] Required                   [ ] Disabled  
-  | - Kerberos                     [x] Alternative  [ ] Required                   [ ] Disabled
-Identity Provider Redirector       [x] Alternative  [ ] Required                   [ ] Disabled  
-Authenticate with Keycloak         [x] Alternative  [ ] Required  [ ] Conditional  [ ] Disabled  
-  | - Username                     [ ] Alternative  [x] Required                   [ ] Disabled
-  | - 1st                          [ ] Alternative  [x] Required  [ ] Conditional  [ ] Disabled
-       | - Password                [x] Alternative  [ ] Required                   [ ] Disabled
-  | - 2nd                          [ ] Alternative  [x] Required  [ ] Conditional  [ ] Disabled
-       | - OTP                     [x] Alternative  [ ] Required                   [ ] Disabled
-       | - WebAuthn                [x] Alternative  [ ] Required                   [ ] Disabled
-```
+Keycloak should have the same built in flows as is currently the case, except adapted to the new logic.
+For the browser flow:
+![Default Browser](img/multi-factor-admin-and-step-up/DefaultBrowser.png)
 
-And for the Direct Grant the flow would be:
-```
-Auth type                         | Requirement
------------------------------------------------------------------------------------------------
-Username Validation                [ ] Alternative  [x] Required                   [ ] Disabled
-Password                           [ ] Alternative  [x] Required                   [ ] Disabled
-2nd Factor                         [ ] Alternative  [x] Required  [ ] Conditional  [ ] Disabled
-  | - OTP                          [x] Alternative  [ ] Required                   [ ] Disabled
-  | - Other                        [x] Alternative  [ ] Required                   [ ] Disabled
-```
+For the direct grant flow:
+![Default Direct Grant](img/multi-factor-admin-and-step-up/DefaultBrowser.png)
```

Comment:
The screenshot for "Direct Grant" contains browser flow. Looks like a mistake?

Reply:
yes, I've pushed a correction